doc/user/content/security/self-managed/authentication.md

```diff
@@ -148,6 +148,25 @@ When SASL authentication is enabled:
 This hybrid approach provides maximum security for SQL connections while maintaining
 compatibility with web-based tools.
 
+## Deploying authentication
+
+Using the configured Materialize CR YAML file, we recommend rolling out the changes by adding the following fields:
+```yaml
+spec:
+  ...
+  requestRollout: <SOME_NEW_UUID> # Generate new UUID for rollout
+  forceRollout: <SOME_NEW_UUID> # Rollout without requiring a version change
+  # Tears down the prior version and restarts the current Materialize instance
+  rolloutStrategy: ImmediatelyPromoteCausingDowntime
+```
+
+For more information on rollout configuration, view our [installation overview](/installation/#rollout-configuration).
+
+{{< warning >}}
```

Comment:
Could we repeat this warning in both places above where we talk about setting `authenticatorKind`? 

It's important enough that I think repeating in throughout (as people hop onto just their interested section)